examples/dynamodb/versions.tf

```diff
@@ -1,4 +1,10 @@
 
 terraform {
-  required_version = ">= 0.12"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.70.0"
```

Comment:
Do we plan to upgrade this in a multi-step approach? I am asking because there is already a newer, major release of the AWS Provider.

Reply:
I just discussed it with @titanlien - yes, we will update it gradually

Reply:
Yes, I would like to split newer version into coming PR. This PR is preparing examples code to verify the v4.* changes